glustercli/cmd/volume.go

```diff
@@ -330,9 +330,18 @@ func volumeInfoHandler2(cmd *cobra.Command, isInfo bool) error {
 		}
 	} else {
 		table := tablewriter.NewWriter(os.Stdout)
-		table.SetHeader([]string{"ID", "Name"})
+		table.SetAlignment(tablewriter.ALIGN_LEFT)
```

Comment:
suggestion: can we have Align to center ```table.SetAlignment(tablewriter.ALIGN_CENTER)``` 
to have better view.

Reply:
I followed the existing convention which is left:

```sh
$ grep -nR "SetAlignm" glustercli/
glustercli/cmd/georep.go:483:		table.SetAlignment(tablewriter.ALIGN_LEFT)
glustercli/cmd/georep.go:511:			table.SetAlignment(tablewriter.ALIGN_LEFT)
```

I tried out your suggestion of center alignment and this is what it looks like:

```sh
$ glustercli volume list
+--------------------------------------+-------------+-----------------------+---------+-----------+--------+
|                  ID                  |    NAME     |         TYPE          |  STATE  | TRANSPORT | BRICKS |
+--------------------------------------+-------------+-----------------------+---------+-----------+--------+
| 94e8d6de-f3dc-46e6-a062-72a64eed9125 |    test     |      Distribute       | Started |    tcp    |   1    |
| e8e72440-025f-4cf0-af0a-d2d6306fba70 | testdistrep | Distributed-Replicate | Created |    tcp    |   4    |
| 3f40f5e6-a8a8-438d-9154-a5799080f0fc |   testrep   |       Replicate       | Stopped |    tcp    |   3    |
+--------------------------------------+-------------+-----------------------+---------+-----------+--------+
```

Left alignment looks better to me :) 

The name and type field doesn't look as good but transport and bricks fields look better. My vote goes for left/right alignment as most standard unix tools' output is never center aligned:

```sh
[ppai@gd2-1 glusterd2]$ df -h | head -n 5
Filesystem               Size  Used Avail Use% Mounted on
devtmpfs                 2.0G     0  2.0G   0% /dev
tmpfs                    2.0G     0  2.0G   0% /dev/shm
tmpfs                    2.0G  840K  2.0G   1% /run
tmpfs                    2.0G     0  2.0G   0% /sys/fs/cgroup

[ppai@gd2-1 glusterd2]$ lsof | head -n 5
COMMAND     PID   TID    USER   FD      TYPE             DEVICE  SIZE/OFF       NODE NAME
systemd       1          root  cwd   unknown                                         /proc/1/cwd (readlink: Permission denied)
systemd       1          root  rtd   unknown                                         /proc/1/root (readlink: Permission denied)
systemd       1          root  txt   unknown                                         /proc/1/exe (readlink: Permission denied)
systemd       1          root NOFD                                                   /proc/1/fd (opendir: Permission denied)

[ppai@gd2-1 glusterd2]$ ps aux | head -n 5
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  0.0  0.2 158160 11592 ?        Ss   12:46   0:01 /usr/lib/systemd/systemd --switched-root --system --deserialize 24
root         2  0.0  0.0      0     0 ?        S    12:46   0:00 [kthreadd]
root         4  0.0  0.0      0     0 ?        I<   12:46   0:00 [kworker/0:0H]
root         6  0.0  0.0      0     0 ?        I<   12:46   0:00 [mm_percpu_wq]

[ppai@gd2-1 glusterd2]$ ls -l | head -n 5
total 15320
drwxrwxr-x.  2 ppai ppai      116 Jun 29 17:27 build
drwxrwxr-x.  2 ppai ppai       26 Jun  6 21:31 constants
-rw-rw-r--.  1 ppai ppai    18092 Mar 21 19:09 COPYING-GPLV2
-rw-rw-r--.  1 ppai ppai     7651 Mar 21 19:09 COPYING-LGPLV3
```